app/scripts/helpers/input-transform.helpers.js

```diff
@@ -0,0 +1,60 @@
+// test/ampersand //see -> test&/see
+export function rawToEscapedContent(rawText, glyphs = {}) {
+	return arrayToEscapedContent(contentToArray(rawText), glyphs);
+}
+
+// ['t', 'e', 's', 't', '/ampersand ', '//', 's', 'e', 'e'] -> test&/see
+export function arrayToEscapedContent(textArray, glyphs = {}) {
+	return textArray.map((letter) => {
+		if(letter === '//') {
+			return '/';
+		}
+		if(letter.startsWith('/')) {
+			const glyphName = letter.slice(1).trim();
+
+			const [glyph] = Object.values(glyphs).find(([g]) => {
+				return glyphName === g.src.glyphName;
+			}) || [];
+
+			if(glyph) {
+				return String.fromCharCode(glyph.src.unicode);
+			}
+			return '';
+		}
+		return letter;
+	}).join('');
+}
+
+// ['t', 'e', 's', 't', '/ampersand ', '//', 's', 'e', 'e'] -> test&//see
+export function arrayToRawContent(textArray) {
+	return textArray.join('');
+}
+
+// test&/see -> ['t', 'e', 's', 't', '&', '//', 's', 'e', 'e']
+export function contentToArray(rawText) {
+	const buffer = [];
+	let slashBefore = false;
+
+	rawText.split('').forEach((letter) => {
+		if(slashBefore) {
+			let command = buffer.pop();
+
```

Comment:
seems really innefficient to push and pop letter on the buffer. You can have a string that you add to when slashbefore is true and push that when you encounter space or slash.